pimpmycause/pimpmycause/templates/profiles/detail.html

```diff
@@ -0,0 +1,87 @@
+{% extends "layout.html" %}
+
+{% block title %} Profile {% endblock %}
+
+{% load static %}
+
+    {% block main %}
+    <section class="page-register pnl row">
+        <div class="content-wrap">
+            <h1 class="pnl__title">Profile</h1>
+                <div class="medium-12 large-12 small-10 large-centered small-centered columns">
+                        <div class="row">
+                            <div class="large-3 medium-4 small-10 small-centered medium-uncentered columns">
+
+                                {% if not profile_update_form.image.value %}
```

Comment:
I think this should work without checkign for value, if a field is not set `profile_update_form.image` should evaluate false